README.md

```diff
@@ -1,14 +1,14 @@
 # sbt-http4s-org
 
 This project is intended to set up sensible defaults and standards for projects under http4s.org.
-If it's useful outside http4s.org, it probably should be pushed further upstream into [sbt-spiewak](https://github.com/djspiewak/sbt-spiewak/).
+If it's useful outside http4s.org, it probably should be pushed further upstream into [sbt-typelevel](https://github.com/typelevel/sbt-typelevel).
 
 ## Installing it
 
 The plugin is deployed to Sonatype.
 
 ```scala
-addSbtPlugin("org.http4s" % "sbt-http4s-org" % http4sOrgV)
+addSbtPlugin("org.http4s" % "sbt-http4s-org" % "@VERSION@")
```

Comment:
This is ugly in the GitHub README but it becomes useful information on the live site.